src/main/java/com/acoustic/SpringPolandSalaryCalculator/controller/SalaryCalculatorController.java

```diff
@@ -0,0 +1,124 @@
+package com.acoustic.SpringPolandSalaryCalculator.controller;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Comparator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.acoustic.SpringPolandSalaryCalculator.calculatorservice.SalaryCalculatorService;
+import com.acoustic.SpringPolandSalaryCalculator.entity.DataSalaryCalculator;
+import com.acoustic.SpringPolandSalaryCalculator.jobcategories.JobCategoriesConfigurationProperties;
+import com.acoustic.SpringPolandSalaryCalculator.service.DataSalaryCalculatorRepository;
+import lombok.RequiredArgsConstructor;
+
+
+@RestController
+@RequiredArgsConstructor
+@Validated
+public class SalaryCalculatorController {
+
+    public static final int MINIMUM_GROSS = 2000;
+    private final DataSalaryCalculatorRepository dataSalaryCalculatorRepository;
+    private final List<SalaryCalculatorService> salaryCalculatorServices;
+    private final JobCategoriesConfigurationProperties jobCategoriesConfigurationProperties;
+
+
+    @GetMapping("/getJobTitles/{departmentName}")
+    public List<String> getJobTitles(
+            @PathVariable
+            String departmentName) {
+        return this.jobCategoriesConfigurationProperties.getJobDepartmentAndTitles().get(departmentName);
+    }
+
+    @GetMapping("/getJobDepartments")
+    public Set<String> getDepartmentName() {
```

Comment:
Why is the difference in returned type? departments cannot be duplicated but job titles can?